pallets/pallet-lottery/Cargo.toml

```diff
@@ -0,0 +1,78 @@
+[package]
+authors = ["Adam Reif"]
+edition = "2021"
+name = "pallet-lottery"
+version = "0.1.0"
```

Comment:
version number

Reply:
And more metadata about this pallet

Reply:
the code was originally forked from their 0.9.27 codebase, updated